tcpdump.c

```diff
@@ -2372,6 +2398,7 @@ static void
 dump_packet_and_trunc(u_char *user, const struct pcap_pkthdr *h, const u_char *sp)
 {
 	struct dump_info *dump_info;
+    int dlt;
```

Comment:
Please use 8-position indentation (tabs or 8 spaces - note that on the operating system where this was first developed, tab stops were every 8 spaces), as is done in the rest of the file.